sources/packages/api/src/utilities/application.ts

```diff
@@ -0,0 +1,27 @@
+import { Application } from "../database/entities";
+import {
+  COE_DENIED_REASON_OTHER_ID,
+  PIR_DENIED_REASON_OTHER_ID,
+} from "../utilities";
+
+/**
+ * Gets PIR denied reason
+ * @param applicationDetails application Object.
+ * @returns PIR denied reason.
+ */
+export function getPIRDeniedReason(application: Application): string {
+  return application.pirDeniedReasonId?.id === PIR_DENIED_REASON_OTHER_ID
+    ? application.pirDeniedOtherDesc
+    : application.pirDeniedReasonId?.reason;
+}
+
+/**
```

Comment:
Just a suggestion. The file name could be something like application-utils.ts.

Reply:
since the file is inside `utils/` folder, IMO, `utilities/application.ts` doesn't seems to be a problem. but in order to follow the consistency renamed the file

Reply:
Yeah it was a suggestion alone because it was inside utilities package. Thanks for making the change.